winpr/include/winpr/wtsapi.h

```diff
@@ -116,7 +116,7 @@ typedef struct tagCHANNEL_PDU_HEADER
 #define VCEXPORT  __export
 #endif
 
-typedef VOID VCAPITYPE CHANNEL_INIT_EVENT_FN(LPVOID pInitHandle,
+typedef UINT VCAPITYPE CHANNEL_INIT_EVENT_FN(LPVOID pInitHandle,
```

Comment:
@mfleisz  for wtsapi at least basic functionality still aiming for compatibility
@akallabeth would you be so kind and the PR?